react/features/connection-indicator/components/ConnectionIndicator.js

```diff
@@ -6,44 +7,22 @@ import { ConnectionStatsTable } from '../../connection-stats';
 
 import statsEmitter from '../statsEmitter';
 
-declare var $: Object;
 declare var interfaceConfig: Object;
 
-// Converts the percent for connection quality into a string recognized for CSS.
-const QUALITY_TO_WIDTH = [
-
-    // Full (5 bars)
-    {
-        percent: 80,
-        width: '100%'
-    },
-
-    // 4 bars
-    {
-        percent: 60,
-        width: '80%'
-    },
-
-    // 3 bars
-    {
-        percent: 40,
-        width: '55%'
-    },
-
-    // 2 bars
-    {
-        percent: 20,
-        width: '40%'
-    },
-
-    // 1 bar
-    {
-        percent: 0,
-        width: '20%'
-    }
-
-    // Note: we never show 0 bars.
-];
+const INDICATOR_DISPLAY_THRESHOLD = 64;
+const CONNECTION_TO_COLOR_MAP = {
+    0: 'crimson',
+    30: 'indianred',
+    47: 'yellow',
+    [INDICATOR_DISPLAY_THRESHOLD]: 'palegreen',
+    81: 'springgreen',
+    98: 'springgreen'
+};
```

Comment:
I didn't know what colors to use so I just picked some from a red-to-green scale. I also didn't know at what percentages to trigger different behaviors, so I copied the original implementation--https://github.com/jitsi/jitsi-meet/pull/1012.